### PR TITLE
tests/nimble_l2cap: adjust buffer sizes for nrf51

### PR DESCRIPTION
**Pull request by haukepetersen**
Enabling the `ble_nimble` feature for nrf51 boards will most likely let `tests/nimble_l2cap` fail due to missing RAM on those boards. This PR adds a crude enhancement to the test application, to limit the used buffers for those boards. Its not pretty, but should do the job for now...

### CODE REVIEW

File: tests/nimble_l2cap_server/nimble.inc.mk [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# For nRF51-based targets, we need to reduce buffer sizes to make this test fit
+# into RAM
+# Note: as the CPU variable is not set at this point, we manually 'whitelist'
+#       all supported nrf51-boards here
+NRF51_WL := airfy-beacon calliope-mini microbit nrf51dk nrf51dongle nrf6310 \
```
Comment (aabadie) [inline tests/nimble_l2cap_server/nimble.inc.mk]: Maybe rename the variable `BOARDS_NRF51` ?

**Review by aabadie — CHANGES_REQUESTED** [review]
I fine with the proposed change but does it work with these low values ? If not I would just simply blacklist the boards.

**Comment by haukepetersen** [comment]
> I fine with the proposed change but does it work with these low values ?

Yes, the test also works with these lower value. Of course its not that much of a stress test, but flooding 500 byte packets on a l2cap channel is still something useful.

**Comment by haukepetersen** [comment]
applied the variable change and squashed.

**Comment by aabadie** [comment]
Your branch is conflicting with my base branch so I directly cherry-picked your commit instead of merging.

**Pull request closed without merging**